docs/contributing.rst

```diff
@@ -8,46 +8,114 @@ For a more practical guide to the tedana development, please see our
-communication, and we strive for consensus amongst all community members.
-
-.. _Benevolent Dictator for Life (BDFL): https://en.wikipedia.org/wiki/Benevolent_dictator_for_life
-
 Code of conduct
 ```````````````
 
-All ``tedana`` community members are expected to follow our `code of conduct`_
-during any interaction with the project.
+All ``tedana`` community members are expected to follow our code of conduct
+during any interaction with the project. `The full code of conduct is here`_.
 That includes---but is not limited to---online conversations,
 in-person workshops or development sprints, and when giving talks about the software.
 
 As stated in the code, severe or repeated violations by community members may result in exclusion
 from collective decision-making and rejection of future contributions to the ``tedana`` project.
 
-.. _code of conduct: https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md
+.. _The full code of conduct is here: https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md
+
+Scope of tedana
+```````````````
+tedana is a collection of tools, software and a community related to echo time
+(TE) dependent analyses. The umbrella of tedana covers a number of overlapping,
+but somewhat distinct ideas related to multi-echo analysis. This scope covers
+collecting multi-echo data (Acquisition), combining those echoes together
+(Combining), with optional noise removal (Denoising), inspecting the outputs
+(Visualization) and answering multi-echo related questions (Community). In
+general, preprocessed data is pushed through tedana, producing outputs that
+are ready for further analyses. 
+
+Acquisition
+-----------
+
+While the development of multi-echo sequences is far beyond the current scope
+of tedana, the tedana community is committed to providing guidelines on current
+multi-echo implementations. This will include both specific instructions for
+how to collect multi-echo data for multiple vendors as well as details about
+what types of data have been collected thus far. These details are subject to
+change, and are intended to provide users with an idea of what is possible,
+rather than definitive recommendations.
+
+Our focus is on functional MRI, including both magnitude and phase data,
+however we understand that quantitative mapping has the potential to aid in
+data processing. Thus, we believe that some details on non-functional MRI
+acquisitions, such as detailed T2* mapping, may fall within the scope of
+tedana. Acquisition related details can be found in the tedana Documentation.
+
+Combining echoes
+----------------
+
+An early step in processing data collected with multiple echoes is the
+combination of the data into a single time series. We currently implement
+multiple options to combine multi-echo data and will add more as they continue
+to be developed. This is an area of active development and interest.
```

Comment:
For "We currently implement multiple options" should we link to the currently supported options ? One idea would be to link the the `fittype` argument in the CLI, specifically [here](https://github.com/ME-ICA/tedana/blob/91b0379e74a25961b2fc7a31875dbc11fb917b2a/tedana/workflows/tedana.py#L80-L91).

Reply:
Somewhat relatedly, in terms of style : it can be helpful if each new sentence is on a new line (for future diffing and for review comments like these). Not worth going through _everything_ now, but if you're updating content anyway, could we move each sentence to its own line ?

Reply:
Another linking question. If I use the link you suggest, isn't that a permalink to a current version of the code. I could link to `fittype` in https://tedana.readthedocs.io/en/latest/generated/tedana.workflows.t2smap_workflow.html but again I'm not sure if there's a more internal way to make that reference or specifically highlight fit type with the link. @jbteves?

Reply:
I'm not sure that we want to link to help in governance as a rule. I think that people who want options can easily find them in the help text or docs, though I definitely see the rationale in linking those things when referenced outside of governance. I just think it would be strange for someone to look at governance before help text if they want options.

Reply:
That's fine if we don't want to link ! I'm not tied to the idea, and I think your rationale makes sense @jbteves.

@handwerkerd if we did want to link it would probably go to the function, so something like: `:py:meth:`combine.make_optcom``.